### PR TITLE
More healing items, plants

### PR DESCRIPTION
**Pull request by chowdermcarthor**
## Description
Re-adds super stims, healing poultice, wasteland harvestable plants, joshua trees, cacti.

## Changelog (neccesary)
:cl:
add: Added back in wasteland harvestable plants.
add: Added back in super stims and healing poultice.
:add: Added in cacti and joshua trees.
/:cl:

### CODE REVIEW

**Comment by chowdermcarthor** [comment]
Note; I didn't wana fuck with the map but there's no good way I could think of to add in cave fungus to random spawns, as it'd spawn within caves. However considering the amount and location and -contents- of the caves in the map right now, such a thing probably requires more discussion.
Also will probably need to tweak the amount of plant spawns - just, wasteland feels more barren than it did in any of the games, especially NV.

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -85,6 +85,23 @@
...
 
+/obj/structure/flora/tree/joshua
+	name = "joshua tree"
+	desc = "A tree named by mormons, who said it's branches mimiced the biblical Joshua, raising his hands in prayer."
+	icon = 'icons/obj/flora/deadtrees.dmi'
+	icon_state = "joshua_1"
+
+/obj/structure/flora/tree/joshua/Initialize()
+	..()
+	pixel_x = 0
```
Comment (LetterN) [inline code/game/objects/structures/flora.dm]: why is that there?

File: code/game/objects/structures/wasteplants.dm [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	if(has_plod)
+		user.put_in_hands(new produce)
+		user << "<span class='notice'>You take [produce] from [src].</span>"
+		has_plod = !has_plod
+		regrow()
+	update_icon()
+
+/obj/structure/flora/wasteplant/proc/regrow()
+	if(!has_plod)
+		spawn(5000)
```
Comment (LetterN) [inline code/game/objects/structures/wasteplants.dm]: make sure `has_plod = !has_plod` and `update_icon()` is on the same level as `sleep(...)`
```suggestion
		sleep(5000)
```

File: code/game/objects/structures/wasteplants.dm [hunk 3]
```diff
@@ -0,0 +1,58 @@
+/obj/structure/flora/wasteplant
+	name = "wasteland plant"
+	desc = "It's a wasteland plant."
+	icon = 'icons/obj/flora/plants.dmi'
+	anchored = 1
+	density = 0
+	var/has_plod = 1
```
Comment (LetterN) [inline code/game/objects/structures/wasteplants.dm]:
```suggestion
	var/has_plod = TRUE
```

File: code/game/objects/structures/wasteplants.dm [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+	var/obj/item/reagent_containers/food/snacks/grown/produce = null
+
+/obj/structure/flora/wasteplant/attack_hand(mob/user)
+	if(has_plod)
+		user.put_in_hands(new produce)
+		user << "<span class='notice'>You take [produce] from [src].</span>"
+		has_plod = !has_plod
```
Comment (LetterN) [inline code/game/objects/structures/wasteplants.dm]:
```suggestion
		has_plod = FALSE
```

File: code/game/objects/structures/wasteplants.dm [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+	icon = 'icons/obj/flora/plants.dmi'
+	anchored = 1
+	density = 0
+	var/has_plod = 1
+	var/obj/item/reagent_containers/food/snacks/grown/produce = null
+
+/obj/structure/flora/wasteplant/attack_hand(mob/user)
+	if(has_plod)
+		user.put_in_hands(new produce)
+		user << "<span class='notice'>You take [produce] from [src].</span>"
```
Comment (LetterN) [inline code/game/objects/structures/wasteplants.dm]:
```suggestion
		to_chat(user, "<span class='notice'>You take [produce] from [src].</span>")
```

File: code/game/objects/structures/wasteplants.dm [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+		has_plod = !has_plod
+		regrow()
+	update_icon()
+
+/obj/structure/flora/wasteplant/proc/regrow()
+	if(!has_plod)
+		spawn(5000)
+			has_plod = !has_plod
```
Comment (LetterN) [inline code/game/objects/structures/wasteplants.dm]: make it `has_plod = TRUE` instead of !has_plod`

**Pull request merged**